app/ide-desktop/common/src/text/english.json

```diff
@@ -159,9 +159,13 @@
   "upgradeToUseCloud": "Upgrade your plan to use Enso Cloud.",
 
   "me": "Me",
+  "or": "or",
   "login": "Login",
   "register": "Register",
   "upgrade": "Upgrade",
+  "startTrial": "Start Trial",
+  "trialDescription": "Start a $0-days free trial",
```

Comment:
`days` -> `day` ([see stack exchange](https://ell.stackexchange.com/questions/114437/which-one-is-right-30-days-or-30-day))

Reply:
TIL